### PR TITLE
Use the absolute path to shell

### PR DESCRIPTION
**Pull request by koln67**
The path to shell requires the / to be accessed

### CODE REVIEW

**Comment by dorileo** [comment]
as we talked offline this PR (and the previous one adding this) is wrong. adding a hold to make sure it doesn't get merged.

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ presubmits:
       - image: gcr.io/compute-image-tools/daisy:latest
```
Comment (dorileo) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml:12]: I guess we also want to change the image, right?

Comment (koln67) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml]: done

**Pull request merged**